cmd/wallet.go

```diff
@@ -68,6 +71,70 @@ var depositsCommand = cobra.Command{
 	},
 }
 
+var sweepCommand = cobra.Command{
+	Use:              "sweep",
```

Comment:
Can we name this command `propose-deposit-sweep`? First, this command just submits the proposal and sweeping is purely a wallet action. Second, in the future, we will also have "moved funds sweep" so it would be good to make it specific that we are dealing with "deposit sweep" here.

Reply:
Maybe `propose-sweep` to keep it shorter?

Reply:
Remember that we will also have "propose moved funds sweep". Unfortunately, we must be specific here and make the command a bit longer.

Reply:
If there will be more we could make the sub-commands nested, e.g.:
```
deposits list
deposits sweep propose
deposits sweep bump-fee
deposits redeem

redemptions list

moved-funds list
moved-funds sweep propose
moved-funds sweep bump-fee
```

But for now, I updated it the way that was proposed: 31b7d3034049361fa6b686f14fcb8920655556b1

Reply:
Yeah, good idea regarding the sub-commands!